homeassistant/components/thermostat/zwave.py

```diff
@@ -51,6 +51,11 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
             if DEVICE_MAPPINGS[specific_sensor_key] == WORKAROUND_IGNORE:
                 _LOGGER.debug("Remotec ZXT-120 Zwave Thermostat, ignoring")
                 return
+    if not (value.node.get_values_for_command_class(
```

Comment:
An empty dict is already falsey while a dict with any info in it is true. So no need to call `items()` on it.

```
› python3
Python 3.4.3 (default, Aug 11 2015, 08:57:25)
[GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.53)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> bool({})
False
>>> bool({'hello': 1})
True
```